sourcemash/api/feeds.py

```diff
@@ -10,6 +10,13 @@
 
 from sourcemash.models import Feed, UserItem, Item
 
+import os
+from rq import Queue
+from worker import create_worker
+from worker.scraper import scrape_articles_only
+
+conn = create_worker(os.environ.get("APP_CONFIG_FILE") or "development")
```

Comment:
It's in the worker's config and I'm not sure if we want to move it into sourcemash's config too. Thoughts?